test/integration/node_pool/controls/kubectl.rb

```diff
@@ -73,13 +73,13 @@
         end
 
         it "has the expected taints" do
-          expect(taints).to include(
+          expect(taints).to eq([
```

Comment:
This appears to have been rebased incorrectly; #200 changed this to `include`.

Reply:
no idea how that happened. It's fixed now.